src/domain/common/utils/deployments.ts

```diff
@@ -0,0 +1,118 @@
+  getMultiSendDeployments as _getMultiSendDeployments,
+  getProxyFactoryDeployments as _getProxyFactoryDeployments,
+  getSafeL2SingletonDeployments as _getSafeL2SingletonDeployments,
+  getSafeSingletonDeployments as _getSafeSingletonDeployments,
+} from '@safe-global/safe-deployments';
+
+/**
+ * Returns a list of official ProxyFactory addresses based on given {@link Filter}.
+ *
+ * @param {string} args.chainId - the chain ID to filter deployments by
+ * @param {string} args.version - the version to filter deployments by
+ *
+ * @returns {Array<`0x${string}`>} - a list of checksummed ProxyFactory addresses
+ */
+export function getProxyFactoryDeployments(args: Filter): Array<`0x${string}`> {
+  return formatDeployments(_getProxyFactoryDeployments, args);
+}
+
+/**
+ * Returns a list of official L1 singleton addresses based on given {@link Filter}.
+ *
+ * @param {string} args.chainId - the chain ID to filter deployments by
+ * @param {string} args.version - the version to filter deployments by
+ *
+ * @returns {Array<`0x${string}`>} - a list of checksummed L1 singleton addresses
+ */
+export function getSafeSingletonDeployments(
+  args: Filter,
+): Array<`0x${string}`> {
+  return formatDeployments(_getSafeSingletonDeployments, args);
+}
+
+/**
+ * Returns a list of official L2 singleton addresses based on given {@link Filter}.
+ *
+ * @param {string} args.chainId - the chain ID to filter deployments by
+ * @param {string} args.version - the version to filter deployments by
+ *
+ * @returns {Array<`0x${string}`>} - a list of checksummed L2 singleton addresses
+ */
+export function getSafeL2SingletonDeployments(
+  args: Filter,
+): Array<`0x${string}`> {
+  return formatDeployments(_getSafeL2SingletonDeployments, args);
+}
+
+/**
+ * Returns a list of official MultiSendCallOnly addresses based on given {@link Filter}.
+ *
+ * @param {string} args.chainId - the chain ID to filter deployments by
+ * @param {string} args.version - the version to filter deployments by
+ *
+ * @returns {Array<`0x${string}`>} - a list of checksummed MultiSendCallOnly addresses
+ */
+export function getMultiSendCallOnlyDeployments(
+  args: Filter,
+): Array<`0x${string}`> {
+  return formatDeployments(_getMultiSendCallOnlyDeployments, args);
+}
+
+/**
+ * Returns a list of official MultiSend addresses based on given {@link Filter}.
+ *
+ * @param {string} args.chainId - the chain ID to filter deployments by
+ * @param {string} args.version - the version to filter deployments by
+ *
+ * @returns {Array<`0x${string}`>} - a list of checksummed MultiSend addresses
+ */
+export function getMultiSendDeployments(args: Filter): Array<`0x${string}`> {
+  return formatDeployments(_getMultiSendDeployments, args);
+}
+
+type Filter = {
+  chainId: string;
+  version: string;
+};
+
+/**
+ * Helper to remap {@link SingletonDeploymentV2} to a list of checksummed addresses.
+ *
+ * @param getDeployments - deployment getters from @safe-global/safe-deployments
```

Comment:
Param types missing

Reply:
Added in 842426ee25749a095fff340f0f5722ee9a5990ea.